csrc/host_ir/lower.h

```diff
@@ -16,14 +16,17 @@ namespace nvfuser {
 
 class HostIrLower {
  public:
-  static bool canLower(Expr* expr);
+  static bool canLower(Expr* expr, bool ignore_inner_resharding = false);
```

Comment:
I'd appreciate a brief code comment on why this flag is needed -- because insert_reshardings and reorder_sharded_axis want different behaviors.